src/src/main.c

```diff
@@ -123,16 +110,13 @@ void loop(void)
 
   errorCheck();
 
-  // writeEEPROM();
+  writeEEPROM();
```

Comment:
This needs to be protected so that it wont write flash everytime. I will take look of the memcmp case which protects wearing the flash

Reply:
At the moment it doesn't write every loop, and only when updateEEPROM is called.  If memcmp works it would be great to just rely on it.

Reply:
Good, i missed that